packages/rancher-logging/overlay/templates/loggings/gke/logging.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.additionalLoggingSources.gke.enabled }}
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Logging
+metadata:
+  name: {{ .Release.Name }}-gke
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
+spec:
+  controlNamespace: {{ .Release.Namespace }}
+  fluentbit:
+    image:
+      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}
+      tag: {{ .Values.images.fluentbit.tag }}
+    inputTail:
+      Tag: "gke"
+      Path: "/var/log/kube-proxy.log"
+      Parser: "syslog"
```

Comment:
is this a valid parser?

Reply:
It's the same `parser` as what EKS uses. A warning appears that it may not be available though.

Reply:
If it is raising a warning we should remove it in both places.  Or try to see if a valid parser exists.

Reply:
I'll remove it from AKS/GKE for now.